backend/app/src/main/kotlin/me/kruemmelspalter/file_spider/backend/renderer/Renderer.kt

```diff
@@ -28,21 +28,30 @@ class Renderer {
         }
         private val markdownRenderer =
             Renderer().tempDir().replace(mapOf("->" to "$\\rightarrow$", "=>" to "$\\Rightarrow$"))
-                .command(10) { "pandoc -f markdown -o out.html ${it.document.id}" }
+                .command(10) { "pandoc -f markdown -o out.html -s --katex='https://cdn.jsdelivr.net/npm/katex@0.16.3/dist/' ${it.document.id}" }
```

Comment:
maybe serve statically so no internet is required?

Reply:
resolved by [4d95ba8](https://github.com/Kruemmelspalter/FileSpider/pull/10/commits/4d95ba8fb8bd92432fb71594e18feadef149ab41)